test/main.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func NewTestLogger() *logrus.Logger {
```

Comment:
should this file be `test/logger.go` instead?

Reply:
weird, it is `test/logger.go` 😉